### ISSUE TITLE
Segfaults since 1.1

### ISSUE THREAD

**Issue by fschwan** [baedert/corebird#407]
At first I'd like to thank you for this great peace of software :)

Since 1.1 I cannot start corebird without a Segfault appearing within 10 seconds.

```
(corebird:5175): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(corebird:5175): GLib-GObject-CRITICAL **: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(corebird:5175): GLib-GObject-CRITICAL **: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[1]    5175 segmentation fault (core dumped)  corebird
```

Any ideas?

**Comment by baedert** [comment]
What's your installed gdk-pixbuf version? 2.31.7? If so, downgrade that to 2.31.6.

**Comment by fschwan** [comment]
Seems ok with 2.31.6. Is this a bug of gdk-pixbuf or just an incompatibility issue?

**Comment by baedert** [comment]
It's a bug in gdk-pixbuf. but has already been fixed upstream, so should be fine as soon as 2.32 hits your distribution repos. The only difference in corebird is that we now use `gdk_pixbuf_animation_load_from_stream_async` in *any* case, not just when loading animations (whcih saves a bunch of complexity for us) and since that function is the broken one, is just always segfaults when loading any inline media, not just when loading gif files.

**Comment by fschwan** [comment]
Ok, just tested with 2.32.1. Its ok too.
Thanks!
